programs/Makefile

```diff
@@ -281,17 +279,17 @@ INSTALL_MAN     ?= $(INSTALL_DATA)
 .PHONY: install
 install: zstd
 	@echo Installing binaries
-	@$(INSTALL) -d -m 755 $(DESTDIR)$(BINDIR)/ $(DESTDIR)$(MANDIR)/
+	@$(INSTALL) -d -m 755 $(DESTDIR)$(BINDIR)/ $(DESTDIR)$(MANDIR)/man1
```

Comment:
Is there an issue inserting `/man1` directly into `$MANDIR` ?
Would you prefer such variable to be called `$MAN1DIR` instead ?

Reply:
Disclaimer: this is my first time messing with a cmake-ish (but it's really not even that) build system; from what I recall from messing with autotools and even much older Makefiles, I've never seen MANDIR have the man? subdir in it -- it's always been $(MANDIR)/man?/